polkadot/xcm/xcm-simulator/fuzzer/src/main.rs

```diff
@@ -155,6 +155,35 @@ fn run_input(xcm_messages: [XcmMessage; 5]) {
 	println!();
 
 	for xcm_message in xcm_messages {
+		// We block Transact messages because they can cause panics that are due to the test
+		// environment, and not XCM
```

Comment:
For example?

Reply:
I just checked, and got the following:
```
Unexpected event: RuntimeEvent::MessageQueue(
    Event::ProcessingFailed {
        ...
        error: ProcessMessageError::Unsupported,
    },
)
```
[in this line of the event processing code](https://github.com/louismerlin/polkadot-sdk/blob/80808d34fa1ae8eef6c30c1d6cc0555aa03596db/polkadot/xcm/xcm-simulator/src/lib.rs#L148) when processing an invalid Transact call.
@ggwpez @franciscoaguirre do you think this should be handled differently, e.g. by allowing Unsupported messages instead of panicking, and then removing the blocklist?

Reply:
We could allow that. The thing is arbitrary Transacts will call any type of extrinsic, with wrong attributes or anything of the sort. We could record which Transacts give an error and which ones don't and why. In that way we would know much more than just by blocklisting them